### DISCUSSION TITLE
目前最強翻譯器

### DISCUSSION THREAD

**Discussion by 109302013evantsao** [1101-GS4719-C/Python-Community-GS4719-C#121] [General]
以前不論是在英文作業或是日文作業的時候，都有想過不知道這句要怎麼翻譯的情況，但google翻譯又翻不出合理的句意
現在有一個翻譯器較DeepL，這是我認為目前最好的翻譯器，能得出最為接近你要的意思，我想這種翻譯器應該也算一種機器學習的應用吧
[DeepL](https://www.deepl.com/translator)

**Comment by xCk27x** [comment]
這超好用的 剛剛試著把一些複雜的文法和片語丟進去 它都能精準的翻譯出原意欸！

**Comment by Jie2222** [comment]
感謝同學實用分享，精準的翻譯真的很棒

**Comment by mesakiyuta** [comment]
感謝同學分享，測試一些日文文章進去，蠻意外的是它不是一個個詞去翻譯，而是以整體文章的意思去翻譯，因此常常會省略主詞及一些字詞，來讓文章比較通順，然而雖然有少詞，整體意思還是有被正確的翻譯出來。

**Comment by ghost** [comment]
這個真的很好用!看論文沒時間的細看原文時候可以用這個快速找自己想看的部分再回去看原文的(沒辦法英文太爛了==
ppt和word可以整個檔案一起翻譯

**Comment by wax75397** [comment]
推個，真的很實用，打全英文報告時都靠DeepL幫我快速建構句子，之後再自己修改一些字詞就好了

**Comment by Jen-Siang** [comment]
推，原來有這麼好用的東西，我之前都用google翻譯翻出一些奇怪字句，剛剛試了一下發現DeepL翻譯的真的好太多了，實用的工具~~

**Comment by enoch-lightning** [comment]
我剛打英文翻譯，覺得DeepL還需要改進。不過，原因可能是大陸用語與我們不同。如果DeepL能加入繁體中文就更好了。我打I love to eat pancakes.應該要翻成「我喜歡吃鬆餅」比較貼切。
<img width="960" alt="DeepL I love to eat pancakes translation" src="https://user-images.githubusercontent.com/86505454/139044420-b372d3e2-47ff-4caa-bcc3-245f26e3cb9f.png">

**Comment by Isongzhe** [comment]
deepl 一次可以翻到5000字，真的很好用，不過我覺得有時候單詞的翻譯會有點過譯

**Comment by 109302003** [comment]
感謝分享，對於我這個日文初學者很有幫助👍

**Comment by YuHengChen54** [comment]
哇以前都不知道這個，Google翻譯真的常常翻不好
